data/hai/hai reveal 4 middle.txt

```diff
@@ -0,0 +1,462 @@
+			choice
+				`	(Walk into the cave.)`
+				`	(Come back later.)`
+					defer
+			`	You walk twenty meters into the cave and pass through an open doorway. There's a variety of medical machinery and some computers in a makeshift base. Some of the people you've rescued are present, with some more humans and Hai you haven't met. A few of them have pulse rifles. A woman walks towards you.`
+			`	"Greetings Captain <last>, my name is Teeneep Goot'r. I'm a civil rights advocate among the Hai, and I'm trying to help your friends with a problem. As you know, they were kidnapped by humans. A few other Hai and humans victims are here; they were rescued by other kind merchants in human space."`
+			`	"My organization researched their problem and discovered some patterns to the disappearances. We've found these kidnappings have been going on for several years. What is strangest is the microtransceivers in most of the kidnapped victims."`
+			choice
+				`	"What patterns have you noticed in the kidnappings?"`
+					goto patterns
+				`	"Tell me about the microtransceivers."`
+					goto microtransceivers
+				`	"How long have the kidnappings been happening?"`
+					goto kidnappings
+			label questions
+			`	"Do you have any other questions?"`
+			choice
+				`	"How do you know all of this?"`
+					goto know
+				`	"Why are you on an uninhabited planet?"`
+					goto uninhabited
+				`	"Is anyone helping you?"`
+					goto anyone
+				`	"What patterns have you noticed in the kidnappings?"`
+					goto patterns
+				`	"Tell me about the microtransceivers."`
+					goto microtransceivers
+				`	"How long have the kidnappings been happening?"`
+					goto kidnappings
+				`	"How can I help you?"`
+					goto help
+			label uninhabited
+			`	Teeneep looks at you as if you were daft, "The kidnappings happen on Hai and human worlds. The kidnappers have no reason to search an uninhabited planet for kidnapping victims. This gives us a safe place to hide any critical people or information."`
+				goto questions
+			label microtransceivers
+			`	A Hai in a medical uniform explains, "The victims each have have thousands of microscale transmitter-receivers in their body, each the size of a single blood cell. We call them microtransceivers. They act together as one big transmitter and receiver, sending and receiving directional messages over great distances."`
+			`	"This technology is clearly not of human or Hai origin. It has some similarities to some microbots used in Hai medicine that are used to seal off ruptures causing internal bleeding, but the construction of these  is significantly beyond our science and some materials involved have properties unlike anything we've ever studied up close. Another race's hand has been involved in this technology, a race that has mastered miniaturized technology and is comfortable with its principles."`
```

Comment:
I believe we should use a semicolon instead of a "-", just for style purposes.

Reply:
I'm down, but we'd need to do a little more editing to make that work (the phrase after a semicolon needs to be a complete sentence).
```suggestion
			`	"This technology is clearly not of human or Hai origin. It has similarities to some microbots used in Hai medicine that are used to seal off ruptures causing internal bleeding, but the construction of these is significantly beyond our science, and some materials involved have properties unlike anything we've ever studied up close. Another race's hand has been involved in this technology; we're looking for one that has mastered miniaturized technology and is comfortable with its principles."
```
This is the best I can think of off the top of my head. I can't figure out a way to preserve the `a race` bit and still make it a complete sentence, which is unfortunate because that bit's cool. Thoughts?